tests/manifold/spherical_manifold_04.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::Volume:       14109.1
```

Comment:
later on when everyone will forget what this test actually check, it will look strange that it tests volume calculation but numerical results are significatly different than analytical. You can quickly test whether you will get exactly the same number with the current precision with `3` refinement steps. Otherwise i would prefer to have a blessed output where the two numbers are the same.